pyproject.toml

```diff
@@ -34,6 +30,10 @@ mypy = "^1.10.1"
 pylint = ">=2.12.0"
 black = "^22.3.0"
 pytest-dotenv = "^0.5.2"
+numpy = [
+    { version = "^1.16.6", python = ">=3.8,<3.11" },
+    { version = "^1.23.4", python = ">=3.11" },
```

Comment:
Why retain the version caps ? imho these should be reverted (https://github.com/databricks/databricks-sql-python/pull/504)

Reply:
I've now incorporated the changes from #504 as well